src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -126,10 +127,16 @@ export module DotNet {
     if (jsObject && typeof jsObject === 'object') {
       cachedJSObjectsById[nextJsObjectId] = new JSObject(jsObject);
 
-      const result = {
+      let result: any = {
```

Comment:
Why change the `const` to `let`? I know the contents of the object changes, but the object identity itself doesn't change, does it?